mittab/libs/outround_tab_logic/pairing.py

```diff
@@ -0,0 +1,112 @@
+import math
+
+from mittab.apps.tab.models import *
+
+from mittab.libs.outround_tab_logic.checks import have_enough_rooms
+from mittab.libs.outround_tab_logic.bracket_generation import gen_bracket
+
+from mittab.libs.tab_logic import have_properly_entered_data
+from mittab.libs import errors
+
+
+def good_to_go(num_teams):
```

Comment:
```suggestion
def is_pairing_possible(num_teams):
```

Or something like that is a better function name imo